torchci/components/GroupJobConclusion.tsx

```diff
@@ -8,14 +8,15 @@ import {
   PinnedTooltipContext,
 } from "pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]]";
 
-enum JobStatus {
+export enum JobStatus {
   Success = "success",
   Failure = "failure",
   Neutral = "neutral",
   Cancelled = "cancelled",
   Timed_out = "timed_out",
   Skipped = "skipped",
   Pending = "pending",
+  AllSkipped = "all_skipped",
```

Comment:
My issue description was imprecise—a "skipped" job is not the same as a job that didn't happen. So this is more accurately like `AllNull` or `AllUndefined` or something